core/src/test/scala/unit/kafka/coordinator/CoordinatorMetadataTest.scala

```diff
@@ -59,155 +54,16 @@ class CoordinatorMetadataTest extends JUnitSuite {
   @Test
   def testAddGroupReturnsPreexistingGroupIfItAlreadyExists() {
     val groupId = "group"
-    val group1 = coordinatorMetadata.addGroup(groupId, "range")
-    val group2 = coordinatorMetadata.addGroup(groupId, "range")
+    val group1 = coordinatorMetadata.addGroup("range", groupId)
```

Comment:
Your addGroup change now takes in (groupId, protocolType). It probably won't change the outcome of the tests, but better for clarity to match up with the intended usage.